src/components/action-bar/action-bar.types.ts

```diff
@@ -0,0 +1,9 @@
+import { MenuItem } from '../menu/menu.types';
+
+export interface ActionBarItem extends MenuItem {
+    /**
+     * Renders the button in the action bar without their labels.
+     * Does not affect the items that are overflown into the overflow menu.
+     */
+    iconOnly?: boolean;
```

Comment:
Should this be set for each individual item or should it be a prop on the component instead?
Also, not sure about the name here. The name of the property refers to the icon, but it controls the visibility of the label. So maybe `displayLabel` would make more sense (with a default value of `true`)? :thinking:

Reply:
Yes, for each individual item. The reason is that we are extending the types from MenuItem `export interface ActionBarItem extends MenuItem {` to be able to send the same array to the overflow menu, and get things rendered nice and tidy there. in MuneItem, the `text` is required. So consumers need to make sure that they provide a text. Which is also good here due to accessibility reasons (we use the provided `text` in the tooltip, which then also is used by ariadescribedby... you know the story).

`icon` is however option. Which is also good for the action bar. 

But the consumer may not want to display the text for all items, only for some of them. Like 👇 
<img width="671" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/60f4eab9-8886-4058-8f36-0efb699d2a0e">

Which is why we added `iconOnly`.

When they go to the overflow menu, the `iconOnly` has no effect, and it shouldn't.

I have no strong opinions about the name. It can be another thing, but I feel this one is just fine.

Reply:
https://github.com/Lundalogik/crm-feature/issues/3452